### DISCUSSION TITLE
I can't creat new repo

### DISCUSSION THREAD

**Discussion by DoaaMosalam** [community/community#78144] [New to GitHub]
When I create a new repository, I see this message
 "this message Owner is being deleted, so new repository cannot be created."
I don't know why and I want a solution to this problem

**Comment by queenofcorgis** [comment]
Hi there!

To help us assist we may need a few more details like where you are trying to create the repository (browser, desktop, terminal, etc) and if you've successfully created a repository before in that location. Thanks!

**Reply by DoaaMosalam** [reply]
When I create a new repository on the Github website, this message appears to me before. As for the repositories that I created before that, when I change a repository and upload it to the repositories that I created before that, they upload normally and there is no problem with them.
